### PR TITLE
please review

### PR DESCRIPTION
**Pull request by ewhite1999**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,49 @@
...
+This site is a git hub clone. You can search for anyone's username to return their profile and repos. The site is fully responsive.
 
-## Available Scripts
+[Netlify site](git-hub-cloned.netlify.app)
 
-In the project directory, you can run:
+## Table of contents
 
-### `npm start`
+- [Installation & Usage](#installation--usage)
```
Comment (capriceebanks) [inline README.md:9]: I like how the links take you directly to the sections. Nice and convenient feature

File: README.md [hunk 2]
```diff
@@ -1,70 +1,49 @@
...
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Installation & Usage
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- Clone or download the repo.
+- run `npm install` to install the test dependencies.
+- run `npm run dev` to launch the dev build
```
Comment (capriceebanks) [inline README.md:20]: I know it's already been placed in slack but change this to `npm start` please and thanks

File: README.md [hunk 3]
```diff
@@ -1,70 +1,49 @@
-# Getting Started with Create React App
+# LAP 1 Debugging Assignment
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This site is a git hub clone. You can search for anyone's username to return their profile and repos. The site is fully responsive.
 
-## Available Scripts
+[Netlify site](git-hub-cloned.netlify.app)
```
Comment (capriceebanks) [inline README.md:5]: Well done for having your app deployed

**Review by bojin11 — COMMENTED** [review]
I like how you made it only show issues and forks when the user has got a issue or a fork. The README has good explanations of the project with screenshots. You have done lots of tests, and they are all passing. WELL DONE for that! I liked the page design too. Congratulations for the good completion of the code challenge project!!

File: README.md [hunk 4]
```diff
@@ -1,70 +1,49 @@
...
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+## Wins & Challenges
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+### Wins
 
-## Learn More
+- The site is fully responsive (try changing your screen size)
+- The site is deployed to netlify
```
Comment (capriceebanks) [inline README.md:45]: Big win

File: src/App.js [hunk 5]
```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
-
+import React from "react";
+import { RepoPage } from "./Pages";
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
```
Comment (capriceebanks) [inline src/App.js:8]: Didn't realise you could make the logo dynamic... you learn something new everyday

File: src/Components/Profile/Profile.spec.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
+/**
```
Comment (capriceebanks) [inline src/Components/Profile/Profile.spec.js:1]: TESTS TOO?? Yeah well done guys lol

File: src/Components/Profile/index.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
```
Comment (capriceebanks) [inline src/Components/Profile/index.js:1]: Used react well and integrated tailwind efficiently enough to produce both a desktop and mobile app 👏

**Review by capriceebanks — COMMENTED** [review]
Overall guy, I hated the pro- no I'm joking LOL. Honestly, you guys did really well; you made the distinction between the mobile and desktop app smooth as well as deploying your app onto netlify. Really like it guys, just change the README and you're good 👏

**Pull request merged**